### PR TITLE
Feature/jc 607 book catalog

### PR DESCRIPTION
**Pull request by ResicHub**

### CODE REVIEW

File: apps/vanilla/src/global/base.css [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+}
+
+/* For mobile phones: */
+[class*="col_"] {
+  width: 100%;
+}
+
+@media only screen and (min-width: 689px) {
+  /* For tablets: */
+  .col_s_1 { width: 8.33%; }
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/global/base.css:12]: As a rule, our designers do not use a 12-column grid.

File: apps/vanilla/src/global/fonts.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+html {
+  font-family: Inter, sans-serif;
+}
+
+[class*="font-style_"] {
+  color: var(--text-color);
+  line-height: 24px;
+}
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/global/fonts.css:8]: This is an interesting trick, this is the first time I’ve seen someone use it.

But, usually each font size has its own line height

File: apps/vanilla/src/global/fonts.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+html {
+  font-family: Inter, sans-serif;
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/global/fonts.css:2]: I thought you should use `Roboto` font

File: apps/vanilla/src/pages/index.css [hunk 4]
```diff
@@ -1,5 +1,231 @@
...
+  --page-min-width: 360px;
+  --page-max-width: 1200px;
+  --header-height: 64px;
+  --filters-control-height: 48px;
+  --books-header-height: 72px;
+  --book-card-min-width: 148px;
+  --book-card-height: 307px;
+  --card-image-height: 250px;
+  --footer-height-large: 180px;
+  --footer-height-medium: 124px;
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/pages/index.css:13]: I don't think variable widths will be useful.

File: apps/vanilla/src/pages/index.css [hunk 5]
```diff
@@ -1,5 +1,231 @@
...
+  margin-top: -24px;
+}
+
+.books {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.books-header {
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/pages/index.css:47]: We usually use [BEM](https://en.bem.info/)

File: apps/vanilla/src/pages/index.css [hunk 6]
```diff
@@ -1,5 +1,231 @@
...
+}
+
+option[value=""] {
+  display: none;
+}
+
+option {
+  color: var(--text-color);
+  background-color: var(--input-background-color);
+}
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/pages/index.css:129]: Looks like styles are reusable components. Let's make a separate file for each

File: apps/vanilla/src/pages/index.html [hunk 7]
```diff
@@ -1,16 +1,95 @@
...
+  <body class="page">
+		<header class="header">
+			<label class="font-style_logo">📚 Books app</label>
+		</header>
+    <main class="main">
+			<div class="filters col_3 col_s_12">
+				<label class="font-style_subtitle">filters</label>
+				<div class="filters-controls">
+					<input class="filters-control font-style_body" type="search" placeholder="e.g. “Refactoring”" name="search">
+					<select class="filters-control select font-style_body" required>
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/pages/index.html:19]:
```suggestion
					<select class="filters-control select font-style_body">
```

Most often, filters are not required, but if you need a required value, it is better to remove the option to select an empty value

File: apps/vanilla/src/pages/index.html [hunk 8]
```diff
@@ -1,16 +1,95 @@
...
+						<div class="books-card">
+							<img src="images/books/domain-driven-design.png"/>
+							<label class="font-style_body-small">Domain-Driven Design</label>
+						</div>
+						<div class="books-card">
+							<img src="images/books/implementing-domain-driven-design.png"/>
+							<label class="font-style_body-small">Implementing Domain-Driven Design</label>
+						</div>
+						<div class="books-card">
+							<img src="images/books/sem-navikov-visokoeffectivnich-ludey.png"/>
```
Comment (andrey-gazzaev) [inline apps/vanilla/src/pages/index.html:74]: You forgot the `alt` attribute

**Review by andrey-gazzaev — APPROVED** [review]

File: apps/vanilla/src/global/fonts.css (hunk 3 above)
Comment (ResicHub) [reply apps/vanilla/src/global/fonts.css:2]: At Figma Design I see Inter font
image

File: apps/vanilla/src/pages/index.css (hunk 6 above)
Comment (ResicHub) [reply apps/vanilla/src/pages/index.css:129]: It is ok if I move them to `components.css` file? As well as `.checkbox-container` and `.checkmark`

File: apps/vanilla/src/pages/index.html (hunk 7 above)
Comment (ResicHub) [reply apps/vanilla/src/pages/index.html:19]: I found that using `required` option is the only way how I can change text color at select field for default value.

**Review by khoanguyen-saritasa — APPROVED** [review]

**Pull request merged**